cmem_plugin_base/dataintegration/plugins.py

```diff
@@ -51,6 +52,13 @@ class PluginBase:
 class WorkflowPlugin(PluginBase):
     """Base class of all workflow operator plugins."""
 
+    input_ports: InputPorts
```

Comment:
@robertisele Input Ports are mandatory?

Reply:
Yes, declaring input and output ports are mandatory. But if the user still doesn't declare those it defaults to this:
```
    op.inputPorts shouldBe FlexibleNumberOfInputs()
    op.outputPort shouldBe Some(UnknownSchemaPort)
```